clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -495,8 +497,22 @@ public ApiException build(String message) {
             public ApiException build(String message) {
                 return new OperationNotAttemptedException(message);
             }
-        });
-
+    }),
+    KAFKA_STORAGE_ERROR(56, "Disk error when trying to access log file on the disk.",
```

Comment:
Sure. I will go over all existing request and see if they need to handle KafkaStorageException specifically.